explore/package.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Will this allow us to use dependabot to keep JS dependencies updated?

Reply:
I'd expect so, I'm still a bit undecided on whether to keep two package.json files or only one